adoc/deployment-bootstrap.adoc

```diff
@@ -105,6 +105,30 @@ way to revert this modification. Please choose wisely.
 ====
 
 
+==== Configuring {kube} Services
+
+Inspect the `kubeadm-init.conf` file inside your cluster definition and set extra configuration settings supported by `kubeadm`.
+The latest supported version is {kubeadm_api_version}.
+Later, when you later run `skuba node bootstrap`, `kubeadm` will read `kubeadm-init.conf`
+and will forcefully set certain settings to the ones required by {productname}.
+
+===== Network Settings
+It is important that you adjust the networking settings inside `kubeadm-init.conf` before the bootstrap.
```

Comment:
This is optional and depends on whether the user wants to change the pod and service subnets.